static/css/main.css

```diff
@@ -1,31 +1,13 @@
 body {
 	margin: 0;
 	font-size: 140%;
-	background-color: #333;
-	color: #FFF;
 	font-family: "Helvetica Neue",-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
 }
 
-header:not(.main-header) {
-	height: 2em;
-	padding: 1em;
-	display: flex;
-	align-items: center;
-	background-color: #222;
-	font-size: 1.5em;
-	font-weight: bold;
-	justify-content: space-between;
-}
-
-header .intro {
-	display: flex;
-	flex-direction: row;
-	align-items: center;
-}
-
 h1 {
 	text-align: center;
 	margin-bottom: 1em;
+	color: #1e3242;
```

Comment:
This seems to make the baseline "Turn Law into software" disapear :/

Reply:
😅